src/user/using-the-capture-app.md

```diff
@@ -872,6 +872,21 @@ On the enrollment dashboard, you can view the Person Profile widget.
 
 The Person profile widget is used for viewing key attributes but not for editing. In order to edit the person profile, you must select the `Edit` button to open an edit window. 
 
+### Indicator widget
+
+On the enrollment dashboard, you can view the Indicator widget.
+
+![](resources/images/capture_app/enrollment-dash-indicator-widget-1.png)
+
+The indicator widget displays program indicators that are configured as "Display in form" or through program rules (Displaying text or key/value pairs in "Program indicator widget"). 
+
+#### Empty state
+If there are no indicators available for the enrollment, the widget will be shown with a text saying there are no indicators.
+If there are no program rules or indicators that can possibly display a message, the indicator will not be displayed at all.
```

Comment:
I suggest simplifying to:
`If the current dashboard can't show any indicator output (because it has no related indicators) then the widget is hidden.`